app/pages/reservation/ReservationPage.js

```diff
@@ -166,14 +166,15 @@ class UnconnectedReservationPage extends Component {
       recurringReservations,
       selectedReservations,
       t,
+      isAdmin,
     } = this.props;
 
     const reservationsCount = selectedReservations.length + recurringReservations.length;
     const introText = resource.needManualConfirmation
       ? t('ConfirmReservationModal.preliminaryReservationText', { reservationsCount })
       : t('ConfirmReservationModal.regularReservationText', { reservationsCount });
 
-    return (
+    return (isAdmin && (
```

Comment:
Request to move `isAdmin` check to inside of `RecurringReservationControl` so no extra check have to be performed anymore

Reply:
We don't use `RecurringReservationControl` in many places. So, I guess it is fine.

Reply:
Fix the problem by its root, not just to make it work. Be responsible